docs/user-guide/ldap-configuration.md

```diff
@@ -52,6 +54,29 @@ You can test a URL connection by clicking the `Test connection` button (located
 You can test a URL authentication by clicking the `Test authentication` button (located next to the Bind Credential input) before saving your settings.
 :::
 
+### Group Settings
+
+**Group Searching and Updating**
+1. **Groups DN**:
+   Defines the LDAP tree where groups are located, `for example, ou=groups,dc=iomete,dc=com`. 
+   This is the parent distinguished name (DN) of your LDAP groups.
+2. **Group Object Classes**:
+   A comma-separated list of object classes that identify LDAP group objects, `for instance, groupOfNames`
+3. **Custom User LDAP Filter (Optional)**:
+   Add a custom filter to refine group searches. Use LDAP syntax, `such as (&(objectClass=groupOfNames)(cn={name}))`
+
+**Group and LDAP Attribute Mappings**
+- **Group Attribute (Optional)**:
+   Defines the attributes for LDAP group mapping in IOMETE. This can include:
+
+  ``name: The group's common name (CN), typically mapped to the cn LDAP attribute.
```

Comment:
was this in intended to be in backticks?

![Uploading CleanShot 2024-10-08 at 20.59.04.png…]()